base/applications/drwtsn32/drwtsn32.rc

```diff
@@ -19,3 +19,6 @@
 #ifdef LANGUAGE_EN_US
 	#include "lang/en-US.rc"
 #endif
+#ifdef LANGUAGE_PL_PL
+	#include "lang/pl-PL.rc"
+#endif
```

Comment:
Can you fix also the indentation here and use 4 spaces instead? (please fix also in the english language case :) )

Reply:
Sure :)